doc/developers/advanced_installation.rst

```diff
@@ -376,24 +376,17 @@ Testing
 Testing scikit-learn once installed
 -----------------------------------
 
-Testing requires having the `nose
-<https://nose.readthedocs.io/en/latest/>`_ library. After
+Testing requires having the `pytest
+<https://docs.pytest.org>`_ library. After
 installation, the package can be tested by executing *from outside* the
 source directory::
 
-    $ nosetests -v sklearn
-
-Under Windows, it is recommended to use the following command (adjust the path
-to the ``python.exe`` program) as using the ``nosetests.exe`` program can badly
-interact with tests that use ``multiprocessing``::
-
-    C:\Python34\python.exe -c "import nose; nose.main()" -v sklearn
+    $ pytest sklearn
```

Comment:
Maybe add the `-v` option to be consistent with `make test-code`?

Reply:
I am not sure what the recommendation should be. I rarely use -v myself and I would argue that it is only useful (if at all) if you only have a log file e.g. on our CIs, to make sure that a particular test was run or succeeded or not.

Reply:
I see your point. Personally, I frequently use `-sv` to see, when there is something at the stdout, which unit tests it is associated with. But yes, without a log or the `-s` option, `-v` might not be that useful (particularly when there are lots and lots of tests as in scikit-learn)...